builder/build.sh

```diff
@@ -24,7 +24,8 @@ RAW_IMAGE="rpi-raw.img"
 RAW_IMAGE_VERSION="v0.0.5"
 
 # name of the sd-image we gonna create
-IMAGE_NAME="sd-card-rpi.img"
+IMAGE_VERSION=${VERSION:="v0.0.0"}
+IMAGE_NAME="sd-card-rpi-${IMAGE_VERSION}.img"
```

Comment:
Changing the image file name requires to change the release deployment in [.travis.yml](https://github.com/hypriot/image-builder-rpi/blob/master/.travis.yml#L12) too.